Source/Dafny/Plugin.cs

```diff
@@ -13,18 +13,18 @@ public interface Plugin {
 }
 
 public class ConfiguredPlugin : Plugin {
-  private readonly PluginConfiguration configuration;
+  public readonly PluginConfiguration Configuration;
```

Comment:
I believe public fields are avoided in .NET, but I don't know why so I'm fine with this. The only downside I see is that it might surprise .NET developers coming into this codebase.

Reply:
> I believe public fields are avoided in .NET, but I don't know why so I'm fine with this. The only downside I see is that it might surprise .NET developers coming into this codebase.

I made it into a get property